templates/basic/.github/workflows/size.yml

```
@@ -0,0 +1,15 @@
+name: "size"
+on:
+  pull_request:
+    branches:
+      - master
```

```suggestion
on: [pull_request]
```

I don't think it normally matters what branch it is to, even to a `test` branch this would still be relevant/useful. Also main branches are not always named that and increasingly less nowadays.

---

Agree 👍